java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -73,4 +73,91 @@ public interface BitmapBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> bitcount(String key, long start, long end, BitmapIndexType options);
+
+    /**
+     * Return the position of the first bit set to 1 or 0 in a string.
```

Comment:
```suggestion
     * Return the position of the first bit matching the given `bit` value.
```
You also use 'find' or 'search' here.

Reply:
I think we can use (and re-use) the 'search' and 'match' words on this.  I think the Redis.io description is too verbose for our needs here.